### ISSUE TITLE
NCPA precedence trusts initctl over sysv script status

### ISSUE THREAD

**Issue by meabandit** [NagiosEnterprises/ncpa#344]
RHEL 6
-----------------
Recently there was an issue with NCPA results for a particular service that seemed to be ok according to:
```
$ service <name> status
<name>...is running
```
However initctl didn't agree:
```
$ initctl list |grep <name>
<name> stop/waiting
```
The sysv script was the correct status, but NCPA appeared to return the result from initctl.

Also on a related note, the debug log didn't print anything useful in this regard.  Have some indication as to how a service result was arrived at may be useful.

**Comment by jomann09** [comment]
I have a fix for this that I will apply in 2.0.4 - I will definitely add some more debug logging in the future.

**Comment by jomann09** [comment]
Fixed with 70ad22a3fd260a6bad0888b7b5d2c7a6f85d0832
